### PR TITLE
Support to prepend handler

### LINKED ISSUE #587: Add support to prepend handler
Add support to prepend handler. When using this, handler should be added before other handlers. Also, don't forget to update `RunInterface` with new function.

I found simple solution:
```php
public function prependHandler($handler)
{
    $existingHandlers = array_merge([$handler], $this->getHandlers());
    $this->clearHandlers();

    foreach ($existingHandlers as $existingHandler) {
        $this->pushHandler($existingHandler);
    }
}
```

### PR DESCRIPTION
**Pull request by filips123**
Add support to prepend handler. Fix #587.

### CODE REVIEW

File: src/Whoops/RunInterface.php [hunk 1]
```diff
@@ -25,6 +25,15 @@
...
     public function pushHandler($handler);
 
+    /**
+     * Prepends a handler to the begining of the stack
+     *
+     * @throws InvalidArgumentException  If argument is not callable or instance of HandlerInterface
+     * @param  Callable|HandlerInterface $handler
+     * @return Run
+     */
+    public function prependHandler($handler);
```
Comment (staabm) [inline src/Whoops/RunInterface.php]: Changing the interface is a BC break. We need to check whether this is a problem..

Comment (filips123) [reply src/Whoops/RunInterface.php]: Is it OK to add new function but not update the interface?

Comment (filips123) [reply src/Whoops/RunInterface.php]: @denis-sokolov @staabm Is it OK?

**Comment by denis-sokolov** [comment]
For now #587 is closed, I’m closing this, too. If we reconsider in #587, we can revisit this.

**Pull request closed without merging**